specs/serving/overview.md

```diff
@@ -51,11 +51,11 @@ the status section.
 ## Service
 
 A **Service** encapsulates a **Route** and **Configuration** which together
-provide a software component. Service exists to provide a singular abstraction
+provide a software component. Services exists to provide a singular abstraction
```

Comment:
```suggestion
provide a software component. Services exist to provide a singular abstraction
```

Although I'd argue that it's slightly confusing to say "(plural) exists to provide a singular..."